Softeq.XToolkit.WhiteLabel.iOS/AppDelegateBase.cs

```diff
@@ -44,8 +64,20 @@ protected virtual void InitializeWhiteLabelRuntime()
             OnContainerInitialized(container);
         }
 
+        protected abstract IBootstrapper CreateBootstrapper();
+
         protected virtual void OnContainerInitialized(IContainer container)
         {
+            InitializeNavigation(container);
         }
+
+        protected virtual void InitializeNavigation(IContainer container)
+        {
+            var navigationService = container.Resolve<IPageNavigationService>();
+            navigationService.Initialize(Window.RootViewController);
+            ConfigureEntryPointNavigation(navigationService);
+        }
+
+        protected abstract void ConfigureEntryPointNavigation(IPageNavigationService navigationService);
```

Comment:
It might not be a good idea to force to use IPageNavigationService for initial navigation. IFrameNavigationService could be used in some projects, as well as custom navigation service implementations. Maybe it's better to leave only `protected virtual void InitializeNavigation(...)` in the base class?

Reply:
agree, done